webpack.common.js

```diff
@@ -196,7 +196,7 @@ const config = {
                     path.resolve(__dirname, 'node_modules/flv.js'),
                     path.resolve(__dirname, 'node_modules/is-what'),
                     path.resolve(__dirname, 'node_modules/libarchive.js'),
-                    path.resolve(__dirname, 'node_modules/marked'),
+                    path.resolve(__dirname, 'node_modules/markdown-it'),
```

Comment:
```suggestion
                    path.resolve(__dirname, 'node_modules/markdown-it'),
                    path.resolve(__dirname, 'node_modules/punycode'),
```

Reply:
I just added a param for markdownIt to handle html, seems to work correctly, please test

for your example, it generated
```html
<h1>Disclaimer</h1>
<p>Jellyfin is the volunteer-built media solution that puts you in control of your media. Stream to any device from your own server, with no strings attached. Your media, your server, your way.</p>
<ul>
<li>Jellyfin is the volunteer-built media solution that puts you in control of your media. Stream to any device from your own server, with no strings attached. Your media, your server, your way.</li>
<li>second test line 1<br>
second test line 2</li>
</ul>
<p><a href="https://jellyfin.org/">https://jellyfin.org/</a></p>
<ol>
<li>one</li>
<li>two</li>
<li>three</li>
</ol>
```

Reply:
just a side note, I'm seeing sonarcloud complaining about possible xss etc, looks like a false positive since it still goes through dompurify sanitize

Reply:
ah ok I added punycode to webpack config now